lib/resolver.manifest.js

```diff
@@ -177,13 +177,47 @@ export default class PodletClientManifestResolver {
                 return new utils.AssetJs(obj);
             });
 
-            // Build absolute proxy URIs
-            Object.keys(manifest.value.proxy).forEach((key) => {
-                manifest.value.proxy[key] = utils.uriRelativeToAbsolute(
-                    manifest.value.proxy[key],
-                    outgoing.manifestUri,
-                );
-            });
+            /*
+                START: Proxy backwards compabillity check and handling
+
+                If .proxy is and Array, the podlet are of version 6 or newer. The Array is then an
+                Array of proxy Objects ({ name: 'foo', target: '/' }) which is the new structure
+                wanted from version 6 and onwards so leave this structure untouched. 
+
+                If .proxy is an Object, the podlet are of version 5 or older where the key of the 
+                Object is the key of the target. If so, convert the structure to the new structure
+                consisting of an Array of proxy Objects.
+
+                This check can be removed in version 7 (requires all podlets to be on version 6)
```

Comment:
Is the plan to silently introduce this in v5? Or does this wait until v6? Or did you mean to say v5 where you've said v6 in the comment block explaining the change here?

Reply:
We introduce support for reading old and new data structure in the layout in v5 but we make the podlets NOT serve the new data structure in v5. In v6 we introduce the new data structure. That means that a layout will be able able to consume both v4 and v5 (and newer) podlets and when all layouts are up at v5 we can introduce v6 podlets which has the new data structure. There are in other words a clear upgrade path which is not breaking.

If we introduce the new data structure in podlets in v5, there will be plenty of v4 layouts which will be having issues consuming those.